### PR TITLE
Used UniversalLink and PreviewImage components where needed, to right…

### PR DESCRIPTION
**Pull request by giuliaghisini**
… handle link and images.

### CODE REVIEW

**Comment by giuliaghisini** [comment]
if it were possible, I would give a rather high urgency to this PR. We need it as soon as possible for a client

File: src/components/manage/UniversalLink/UniversalLink.jsx [hunk 1]
```diff
@@ -65,13 +65,25 @@ const UniversalLink = ({
     )?.length > 0;
   const isExternal = !isInternalURL(url) || isBlacklisted;
   const isDownload = (!isExternal && url.includes('@@download')) || download;
+  const isMail = URLUtils.isMail(url) || URLUtils.isMail('mailto:' + url);
+  if (isMail) {
+    url = URLUtils.normaliseMail(url);
+  }
+
+  const isTelephone =
```
Comment (tiberiuichim) [inline src/components/manage/UniversalLink/UniversalLink.jsx:73]: I think this code should be placed inside helpers/Url.js, as it's also used by the Link plugin for draft. https://github.com/plone/volto/blob/2b80b6c3f9811c5fbe069ee25236d41f762a8191/src/helpers/Url/Url.js#L231

**Comment by sneridagh** [comment]
@giuliaghisini could you take care of @tiberiuichim suggestion? I can merge and release as soon it's ready.

File: docs/source/recipes/customizing-components.md [hunk 2]
```diff
@@ -53,61 +53,64 @@ amendments required.
 Locate the `Tags.jsx` file and override this file so that there is a label in front of the tags with: `Tags:`.
 
 ```jsx hl_lines="20"
```
Comment (stevepiercy) [inline docs/source/recipes/customizing-components.md:55]: It appears that this code highlighting directive is not being honored. Let's fix it!

```suggestion
```{code-block) jsx
:emphasize-lines: 20
```

This line appears before the change in the code block, so it should still highlight this line:

```
      Tags:
```

https://github.com/plone/volto/pull/3266/files#diff-b8133ecc81246c0e39589e6ca0088d0b3b443bfb3cda9f055946e6b9965e7933R75

File: docs/source/recipes/customizing-views.md [hunk 3]
```diff
@@ -1,9 +1,9 @@
 ---
 html_meta:
-  "description": "Customize Volto views."
-  "property=og:description": "Customize Volto views."
-  "property=og:title": "Customizing Volto Views"
-  "keywords": "Volto, Plone, frontend, React, views"
+  'description': 'Customize Volto views.'
+  'property=og:description': 'Customize Volto views.'
+  'property=og:title': 'Customizing Volto Views'
+  'keywords': 'Volto, Plone, frontend, React, views'
```
Comment (stevepiercy) [inline docs/source/recipes/customizing-views.md:6]: Please revert the above changes.

**Review by stevepiercy — CHANGES_REQUESTED** [review]
3 reversions and request to fix something that got broken in the conversion to MyST.

**Comment by giuliaghisini** [comment]
@tiberiuichim i've added checkAndNormalizeUrl in Url Helper and used it in UniversalLink, AddLinkForm of AnchorPlugin and in UrlWidget

**Comment by giuliaghisini** [comment]
@sneridagh @tiberiuichim what about to merge and release this fixes?

**Review by stevepiercy — APPROVED** [review]
Thank you!

**Review by tiberiuichim — APPROVED** [review]

**Pull request merged**